src/Stratis.Bitcoin.Features.LightWallet/Blocks/ILightWalletBlockStoreService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Features.LightWallet.Blocks
+{
+    /// <summary>
+    /// This service starts an async loop task that periodically deletes from the blockstore.
+    /// <para>
+    /// If the height of the node's block store is more than <see cref="LightWalletBlockStoreService.MaxBlocksToKeep"/>, the node will 
+    /// be pruned, leaving a margin of <see cref="LightWalletBlockStoreService.MaxBlocksToKeep"/> in the block database.
+    /// </para>
+    /// <para>
+    /// For example if the block store's height is 5000, the node will be pruned up to height 4000, meaning that 1000 blocks will be kept on disk.
+    /// </para>
+    /// </summary>
+    public interface ILightWalletBlockStoreService : IDisposable
```

Comment:
please move this interface to the file with implementation. Remove light wallet from the name. It's not only for light wallet, it's for FN as well

Reply:
I would like to keep interfaces in separate files. I will rename.